Classes/Encoder/UrlEncoder.php

```diff
@@ -1128,7 +1128,7 @@ protected function initializeConfiguration() {
 	protected function isLinkingAcrossDomains() {
 		$result = true;
 
-		foreach (array_reverse($this->tsfe->rootLine) as $page) {
+		foreach ($this->tsfe->rootLine as $page) {
```

Comment:
Explanation for the current code.

`$this->tsfe->rootLine` is reversed. Without `array_reverse`, `foreach` will start from the currently generated page and move up to the tree. With `array_reverse` we start from the root pages (globe level).

So I am trying to understand what will this change actually change, except walking more pages.